pyartcd/pyartcd/pipelines/build_rhcos.py

```diff
@@ -58,184 +52,72 @@ def __init__(self, runtime: Runtime, new_build: bool, ignore_running: bool, vers
 
     def retrieve_auth_token(self) -> str:
         """Retrieve the auth token from the Jenkins service account to use with Jenkins API"""
         # https://github.com/coreos/fedora-coreos-pipeline/blob/main/HACKING.md#triggering-builds-remotely
-
         secret = None
         jenkins_uid = oc.selector('sa/jenkins').objects()[0].model.metadata.uid
         for s in oc.selector('secrets'):
             if s.model.type == "kubernetes.io/service-account-token" and s.model.metadata.annotations["kubernetes.io/service-account.name"] == "jenkins" and s.model.metadata.annotations["kubernetes.io/service-account.uid"] == jenkins_uid:
                 secret_maybe = base64.b64decode(s.model.data.token).decode('utf-8')
-                r = self.request_session.get(
-                    f"{JENKINS_BASE_URL}/me/api/json",
-                    headers={"Authorization": f"Bearer {secret_maybe}"},
-                )
+                r = self.request_session.get(f"{JENKINS_BASE_URL}/me/api/json", headers={"Authorization": f"Bearer {secret_maybe}"},)
                 if r.status_code == 200:
                     secret = secret_maybe
                     break
-
         if secret is None:
             raise Exception("Unable to find a valid Jenkins service account token")
-
         return secret
 
-    @staticmethod
-    def build_parameters(build: Dict[str, List[Dict]]) -> Dict:
-        """Parse the build parameters from the build actions into a more usable dict"""
-        parameters = next((  # first action that has parameters
-            action["parameters"]
-            for action in build["actions"]
-            if "parameters" in action
-        ), [])
-        return {p["name"]: p["value"] for p in parameters}
-
-    @staticmethod
-    def build_url(job: str, number: int) -> str:
-        return f"{JENKINS_BASE_URL}/job/{job}/{number}/"
-
     def query_existing_builds(self) -> List[Dict]:
-        """Check if there are any existing builds for the given version. Returns builds in progress."""
-        builds = []
-        for job in ("build", "build-arch", "release"):
-            response = self.request_session.get(
-                f"{JENKINS_BASE_URL}/job/{job}/api/json?tree=builds[number,description,result,actions[parameters[name,value]]]",
-            )
-            builds_info = response.json()["builds"]
-            for b in builds_info:
-                # build is still running when it has no status
-                if b["result"] is None:
-                    build = dict(**b, job=job, parameters=self.build_parameters(b), url=self.build_url(job, b["number"]))
-                    builds.append(build)
-
-        return [b for b in builds if b["parameters"].get("STREAM") == self.stream]
-
-    @property
-    def stream(self):
-        if self._stream:
-            return self._stream
-
-        # doozer --quiet -g openshift-4.14 config:read-group urls.rhcos_release_base.multi --default ''
```

Comment:
we used to read the config in group and determining if rhcos migrate from old server to new server, now there is only one rhcos server, the stream used for rhcos job parameter should be simplify to directly get the vars from group.